webapp/channels/src/components/integrations/installed_incoming_webhook.tsx

```diff
@@ -102,6 +103,8 @@ export default class InstalledIncomingWebhook extends React.PureComponent<Props>
             );
         }
 
+        const channelDisplayName = channel.display_name ?? 'N/A';
```

Comment:
avoid magic strings. this 'N/A' should be put in the en.jason translation file, like `admin.data_retention.channel_team_counts_empty` does